content/tormobile/tormobile-5/contents.lr

```diff
@@ -1,9 +0,0 @@
-_model: question
```

Comment:
I can make the redirect but then if someone comes along and wants to add to the support section for tormobile, they would likely see `tormobile-4` and make a new `tormobile-5` which the redirect rule would then pickup. I think it would make sense to rename the `tormobile` subsections from something other than `tormobile1,2,3,etc` to something with a more direct name such as `tormobile-guardianproject` or `tormobile-torbrowserandroid` etc

Reply:
I agree with the new folders to be called something nicer, but in this particular case I would also try users not to face a 404 if they try to access a page that they accessed before.
New files can be called nicer names, by choosing a nicer folder name.

Reply:
ended up keeping the original article with another explanation on what happened to Orfox